redirects/14.0.txt

```diff
@@ -54,6 +54,7 @@ applications/inventory_and_mrp/inventory/management/incoming/three_steps.rst app
 applications/inventory_and_mrp/inventory/management/incoming/two_steps.rst applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst                         # /inventory/management/incoming/* --> /management/shipments_deliveries/*
 applications/inventory_and_mrp/inventory/management/replenishment_strategies/virtual_warehouses.rst applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
 applications/inventory_and_mrp/inventory/routes/concepts/inter_warehouse.rst applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
+applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst
 
```

Comment:
This redirect is for the manufacturing app scope, but was placed in the inventory scope. Add a new section for manufacturing below 🙂
```suggestion

# applications/manufacturing

applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst

```